faq/FAQ_Formatting_Guide.txt

```diff
@@ -249,7 +249,7 @@ Good for browsers and text editors, bad for CVS
 
 <p>The purpose of post-processing is to massage an etext into a readable form.  In its journey through two proofreading rounds, the text will have been edited by perhaps hundreds of proofreaders.  The post-processor must standardize the formatting of the book and adjust it to comply with Project's Gutenberg's requirements.  The post-processor must also correct as many errors as possible that have survived both proofreading rounds.  The ultimate goal of post-processing is to create a plain-text etext with consistent formatting throughout, which contains as few errors as possible, and which accurately reflects the intentions of the author.</p>
 
-<p>Post-processors require more experience than ordinary proofreaders.  Because they are preparing the text for uploading to Project Gutenberg, they are the final editors of the text.  Because of this, post-processing is only available for proofreaders who have completed at least <b>400</b> pages in the first and/or second proofreading rounds.  Also, post-processors should be very familiar with the <a href="document.php">Proofreading Guidelines</a> before attempting to post-process.</p>
+<p>Post-processors require more experience than ordinary proofreaders.  Because they are preparing the text for uploading to Project Gutenberg, they are the final editors of the text.  Because of this, post-processing is only available for proofreaders who have completed at least <b>400</b> pages in the first and/or second proofreading rounds.  Also, post-processors should be very familiar with the <a href="proofreading_guidelines.php">Proofreading Guidelines</a> before attempting to post-process.</p>
 <br>
```

Comment:
document.php morphed from the proofing and formatting guidelines before the Great Change into the formatting guidelines, didn't they? In fact, that whole para needs to be rewritten, because access to PP is restricted to those who've done at least 400 F1 pages.

Reply:
Nothing in the source code references this file. We could consider deleting it actually. The last real change in this file was in 2003.

Reply:
Is there a good reason for taking a good look at the file and doing a real update on it? If not, deleting it sounds like a reasonable idea.

Reply:
I don't see a good reason to update it, but I defer to @lhamilton1.

Reply:
I'm fine with deletion. Thanks.